src/server/handlers/github.js

```diff
@@ -137,3 +149,35 @@ const getRequest = (owner, name, token, secret) => {
     }
   };
 };
+
+/*
+ * parse_link_header()
+ *
+ * Parse the Github Link HTTP header used for pageination
+ * http://developer.github.com/v3/#pagination
+ *
+ * Modified by kfenn to return page numbers instead of urls
+ */
+const parseLinkHeader = (header) => {
+  if (header.length == 0) {
+    throw new Error('input must not be of zero length');
+  }
+
+  // Split parts by comma
+  let parts = header.split(',');
+  let links = {};
+  // Parse each part into a named link
+  for (let index=0; index<parts.length; index++) {
+    let section = parts[index].split(';');
+    if (section.length != 2) {
+      throw new Error('section could not be split on ";"');
+    }
+    let number = parseInt(section[0]
+                  .replace(/per_page=(\d+)/, '') // Make sure per_page does not match
+                  .match(/page=(\d+)/)[1]);
```

Comment:
Something involving `url.parse(..., true)` would be neater and would relieve you from fiddling with `per_page`.